config/config.go

```diff
@@ -69,20 +69,20 @@ func (cfg *config) init(env *AppConfig) {
 		cfg.SetUpdate("PhoenixdAuthorization", cfg.Env.PhoenixdAuthorization, "")
 	}
 
-	// set the cookie secret to the one from the env
-	// if no cookie secret is configured we create a random one and store it in the DB
-	cfg.CookieSecret = cfg.Env.CookieSecret
-	if cfg.CookieSecret == "" {
+	// set the JWT secret to the one from the env
+	// if no JWT secret is configured we create a random one and store it in the DB
+	cfg.JWTSecret = cfg.Env.JWTSecret
+	if cfg.JWTSecret == "" {
 		hex, err := randomHex(20)
```

Comment:
Where does this number "20" come from? Is this enough entropy?

> Minimum recommended length: At least 256 bits (32 characters if using base64 or 64 characters if using hexadecimal)

Reply:
Good catch! we should increase it then. 1 hex character = 4 bits = 80 bits entropy.